src/ImageSharp/Configuration.cs

```diff
@@ -75,6 +77,25 @@ public int MaxDegreeOfParallelism
             }
         }
 
+        /// <summary>
+        /// Gets or sets the size of the buffer to use when working with streams.
+        /// Intitialized with <see cref="DefaultStreamProcessingBufferSize"/> by default
+        /// and can accept a minimum value of <see cref="MinStreamProcessingBufferSize"/>.
+        /// </summary>
+        public int StreamProcessingBufferSize
+        {
+            get => this.streamProcessingBufferSize;
+            set
+            {
+                if (value < MinStreamProcessingBufferSize)
+                {
+                    value = MinStreamProcessingBufferSize;
+                }
```

Comment:
We usually throw exception for invalid args. Why a silent change here instead?

Reply:
Basically because I copied StreamWriter which does that for -1 and I thought one less thing to throw. Happy to throw though..

https://source.dot.net/#System.Private.CoreLib/StreamWriter.cs,108

Reply:
I've reverted to throwing for predictability.